src/app/core/components/header/header.component.scss

```diff
@@ -45,3 +45,34 @@
   cursor: pointer;
 }
 
+.user-avatar {
+ 
+  height: 20px;
+  background-color: lightgreen;
+  border-radius: 50%;
+  border: 1px solid grey;
+  padding: 15px;
+  
+}
+
+.listMenu{
+  display: flex;
+  align-items:center
+
+}
+
+.customize+* .mat-menu-panel {
```

Comment:
All code related to mat-menus can be placed in separate file called 'menu.scss' and then it can be imported in 'style.scss'.

Create new 'styles' folder in 'src'.  Then place here 'menu.scss' with next content:

.mat-menu-panel {
    &.customize (selector from 'class' attr of 'mat-menu' component [example below]; I would recommend to rename it to have correct naming across entire app) {
        ... content 
    }
}

Example of 'mat-menu' class:
<mat-menu class="user-menu" ... >
instead of 
<mat-menu backdropClass="customize" ... >